templates/includes/_main_sponsors.html

```diff
@@ -26,6 +52,19 @@
                  alt="Sherpany" style="margin-top: 50px" />
         </a>
     </div>
+    <div class="col-md-3">
+        <a href="https://runcode.io/">
+            <img class="img-responsive center-block" src="{% static 'img/global/supporters/runcode.png' %}"
```

Comment:
@amakarudze It looks like this image is missing

Reply:
Have added the logo, thanks for catching that @marksweb.